src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
-export interface Figure {
+const ERROR_ANY_SIDE_BELOW_OR_IS_ZERO = 'Error, any side length is <= 0';
```

Comment:
English is wrong - it probably should be 
'Error, the length for one of the sides is <= 0'

BTW: you can try grammarly